feat(changelog): cockpit-added-orderable-data-source-for-cockpit 2024-05-24 (#3221)
scaleway/docs-content: changelog/may2024/2024-05-24-cockpit-added-orderable-data-source-for-cockpit.mdx

---

> title: Orderable data source for Cockpit

Reviewer: ```suggestion
title: Data sources are now available in all regions
```

status: added
author:

>     fullname: 'Join the #[xxx] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #observability channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2024-05-24
category: observability
product: cockpit
---

> You can now order and set up as many data sources as you want for external data in Cockpit, available for all regions.

Reviewer: ```suggestion
It is now possible to order and set up as many data sources as you want for your external data in Cockpit, in the Paris, Amsterdam, and Warsaw regions.
```
